src/server/controllers/get.js

```diff
@@ -34,7 +34,11 @@ function get (req, res) {
       fieldname: body.fieldname,
       pathPrefix: `${req.uppy.options.filePath}`,
       pathSuffix: `${encodeURIComponent(id)}`,
-      storage: redisUrl ? redis.createClient({ url: redisUrl }) : null
+      storage: redisUrl ? redis.createClient({ url: redisUrl }) : null,
+      s3: req.uppy.s3Client ? {
```

Comment:
why exactly are we doing this? I don't see it used anywhere in [uploader.js](https://github.com/transloadit/uppy-server/blob/s3%2Fmultipart/src/server/Uploader.js)

Reply:
Oops, did not commit the bit that uses it yet!

Reply:
Just pushed the uploader implementation.